pkg/ccl/streamingccl/streamingest/stream_ingestion_frontier_processor.go

```diff
@@ -418,22 +410,47 @@ func (sf *streamIngestionFrontier) maybeUpdatePartitionProgress() error {
 		return span.ContinueMatch
 	})
 
+	highWatermark := f.Frontier()
```

Comment:
Not your code, but github makes it hard to comment on unmodified lines for whatever reason...
We LoadJob above -- why do we do this every time? It's not needed, is it?

Reply:
Nope its only being used for update, removed it and used  `registry.UpdateJobWithTxn` instead 👍